flink-core/src/main/java/org/apache/flink/api/dag/Transformation.java

```diff
@@ -254,6 +265,34 @@ public ResourceSpec getPreferredResources() {
 		return preferredResources;
 	}
 
+	/**
+	 * Set the managed memory weight which indicates how much this transformation relies
+	 * on managed memory, so that a transformation highly relies on managed memory would
+	 * be able to acquire more managed memory in runtime (linear association). The default
+	 * weight value is 1. Note that currently it's only allowed to set the weight to values
+	 * other than 1 in cases of UNKNOWN resources.
```

Comment:
other than 1? 0?

Reply:
1, because it is the default value

Reply:
adjusted the docs a bit to avoid confusion.